src/App.vue

```diff
@@ -205,8 +205,21 @@ th {
 }
 
 /* SIGN IN/SIGN UP/ BASICALLY FORMS LMAO */
-form {
+.container {
```

Comment:
Need to add as a comment where .container class is used, it's not clear.

Reply:
This was all outdated code

Reply:
you can see that it says outdated on top